src/docsismodem/modems/technicolor_xb7.py

```diff
@@ -94,7 +92,7 @@ def login(self):
         
         try:
             response = self.session.post(self.baseUrl + loginUrl, data=modemAuthentication)
-            if response.status_code == 200:
+            if response.status_code == 200 and "/at_a_glance.jst" not in urlparse(response.url).path:
```

Comment:
@andresp This behavior was exactly what I was looking for. I was prepping a pull request but didn't have time. 

I'll test if this now works out-of-the-box with my XB8

Reply:
Good to hear. Let me know how it goes and I can add the XB8 to the list of supported devices.